src/components/DpModal/DpModal.vue

```diff
@@ -117,68 +89,85 @@ export default {
   },
 
   methods: {
-    getFocusableElements () {
-      const elementList = Array.from(this.$el.querySelectorAll('a, button:not([disabled]), input, textarea, select, details, [tabindex]:not([tabindex="-1"])'))
+    toggle () {
```

Comment:
I prefer more descriptive function names, like `toggleOpen` or `toggleModal`. But if wa are talking about a simple component like Modal, then toggle could also be fine.. but just for me more descriptive name would be better

question: do you use it anywhere? When I click the close button in the upper corner, it does not close the modal